src/components/triangle-boxes/TriangleBoxContainer.js

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import TriangleBoxSmall from './TriangleBoxSmall';
 import TriangleBoxLarge from './TriangleBoxLarge';
 import TriangleBoxArticle from './TriangleBoxArticle';
+import TriangleBoxPartner from './TriangleBoxPartner';
```

Comment:
I think you might be able to repurpose the small or large triangle box here?
This depends a bit on the client feedback though and how they want this to look.
the article box is a hot mess as far as I remember...